model/riscv_sys_regs.sail

```diff
@@ -882,6 +888,24 @@ function is_fiom_active() -> bool = {
     User => (menvcfg.FIOM() | senvcfg.FIOM()) == 0b1,
   }
 }
+
+bitfield Mstateen0 : bits(64) = {
+  // controls to new state introduced by the following small extensions of Priv-1.14-era:
+  PRIV114  : 55
```

Comment:
I would probably do PRIV_1_14, to separate out the 1 and 14 in 1.14

Reply:
Or maybe 1p14 as that's how the notation used in the RISC-V config string works

Reply:
Using 1p14 or 1_14 may end up not match the name that would be in specification and the [riscv-opcodes](https://github.com/riscv/riscv-opcodes/blob/f51aa42dac51c071c516a96648344a50339fa70d/encoding.h#L171).

Reply:
That is an argument for keeping it like it is, but just ambiguously concatenating version numbers like this seems like a bad idea, so I think the inconsistency is probably worth it. I'm not sure we have any other examples of names with version numbers in them. If we want ALLCAPS here maybe 'NAME_V1P14'.

I'm just imagining the inevitable future where we end up with a bunch of versioned identifiers like this, so it might be good to set a convention now.

Reply:
Updated to use PRIV_V1P14.